ote/src/cljs/ote/app/controller/route.cljs

```diff
@@ -260,3 +266,13 @@
                    (every? #(and (time/valid-time? (::transit/departure-time %))
                                  (time/valid-time? (::transit/arrival-time %))) other-stops))))
           trips))
+
+(defn validate-all-steps
```

Comment:
I think a term like "previous" would be better as it does not validate "all" steps.